src/command/mcl.zig

```diff
@@ -11,6 +11,23 @@ var line_accelerations: []u7 = undefined;
 const Direction = mcl.Direction;
 const Station = mcl.Station;
 
+const RegisterLogging = struct {
+    line_index: u8,
+    register_lists: std.EnumArray(RegisterType, bool), // Specify which register to log for each line
+    station_indices: []u8, // Specify the station index to be logged based on axis provided by user
```

Comment:
Similarly, you can use `mcl.Station.Index` here and simplify the name to `stations`